astropy/modeling/tests/test_mappings.py

```diff
@@ -80,7 +81,9 @@ def test_fittable_compound():
     x = np.arange(10)
     y_real = m(x)
     dy = 0.005
-    y_noisy = y_real + np.random.normal(0., dy, x.shape)
+    with NumpyRNGContext(1234567):
+        n = np.random.normal(0., dy, x.shape)
+    y_noisy = y_real + n
```

Comment:
If it works for @astrofrog , it is fine although I still don't see why randomness would make any difference here.

Also, I think mathematically, `assert_allclose(y_fit, y_real, rtol=dy)` should really be `assert_allclose(y_fit, y_real, atol=dy)`?